app/controllers/dashboard/work_form/base_controller.rb

```diff
@@ -19,7 +19,7 @@ def save_and_exit?
 
         def redirect_upon_success
           if save_and_exit?
-            redirect_to dashboard_root_path,
+            redirect_to resource_path(@work_version&.uuid),
```

Comment:
Ditto here. A work version should always have a uuid, unless it's just been created and need to be refreshed.